MAINTAINERS.md

```diff
@@ -1,18 +1,122 @@
 # OpenSearch Maintainers
 
 ## Maintainers
+
+The current maintainers are:
+
 | Maintainer | GitHub ID | Affiliation |
 | --------------- | --------- | ----------- |
 | Abbas Hussain | [abbashus](https://github.com/abbashus) | Amazon |
 | Charlotte Henkle | [CEHENKLE](https://github.com/CEHENKLE) | Amazon |
 | Harold Wang | [harold-wang](https://github.com/harold-wang) | Amazon |
 | Himanshu Setia | [setiah](https://github.com/setiah) | Amazon |
-| Nick Knize | [nknize](https://github.com/nknize) | Amazon | 
+| Nick Knize | [nknize](https://github.com/nknize) | Amazon |
 | Rabi Panda | [adnapibar](adnapibar) | Amazon |
 | Sarat Vemulapalli | [saratvemulapalli](https://github.com/saratvemulapalli) | Amazon |
 | Tianli Feng | [tlfeng](https://github.com/tlfeng) | Amazon |
 | Gopala Krishna Ambareesh | [krishna-ggk](https://github.com/krishna-ggk) |Amazon |
 | Vengadanathan Srinivasan | [vengadanathan-s](https://github.com/vengadanathan-s) | Amazon |
 | Shweta Thareja |[shwetathareja](https://github.com/shwetathareja) | Amazon |
-| Itiyama Sadana | [itiyamas](https://github.com/itiyamas) | Amazon | 
+| Itiyama Sadana | [itiyamas](https://github.com/itiyamas) | Amazon |
 | Daniel "dB." Doubrovkine | [dblock](https://github.com/dblock) | Amazon |
+
+## Versioning
+
+OpenSearch uses semantic versioning based on [Apache’s model](https://commons.apache.org/releases/versioning.html).  
+
+From [Apache’s versioning doc](https://commons.apache.org/releases/versioning.html) : 
+
+A release number is comprised of 3 components: the major release number, the minor release number, and an optional point release number. Here is a sample release number:
```

Comment:
I would like us to say that we will follow semver, and not all of this.

Reply:
I've added a link to https://semver.org/ in the first sentence.  I think the context is useful.